.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,10 +7,9 @@ linked together.
 
 ### Checklist
 
-- [ ] Did you run `npm test`?
+- [ ] Have you added unit and functional tests as appropriate?
 - [ ] Did you update/add documentation for new methods or changed functionality?
-
-Finally, is it ready to be reviewed and merged: yes :white_check_mark: / no :red_circle:
+- [ ] Have you merged the target branch down into this one?
```

Comment:
There's already a message on PRs that haven't done this. Still probably good to have the checkbox.